crates/ruff_python_formatter/README.md

```diff
@@ -231,6 +231,13 @@ additional test cases in `resources/test/fixtures/ruff`.
 
 The full Ruff test suite is slow, `cargo test -p ruff_python_formatter` is a lot faster.
 
+You can check the black compatibility on a number of projects using `scripts/formatter_progress`.
+It will print the similarity index, the percentage of lines that remains unchanged between black's
+formatting and our formatting. You could compute it as the number of neutral lines in a diff divided
+by the neutral plus the removed lines. It also checks for common problems such unstable formatting,
+internal formatter errors and printing invalid syntax. We run this script in CI and you can view the
+results in a PR page under "Checks" > "CI" > "Summary" at the bottom of the page.
```

Comment:
Is there a good resource/reference for users to know whether the numbers are improving or regressing compared to main? I looked at the result at my other PR and wasn't sure against what I should compare the numbers.

Reply:
not really, but i'd rather solve the tracking-numbers-from-CI in general over building more CI-interacting script